packages/appium/lib/cli/extension-command.js

```diff
@@ -486,4 +487,51 @@ export default class ExtensionCommand {
     this.config.installedExtensions[ext].version = version;
     await this.config.write();
   }
-}
+
+  /**
+   * Runs a script cached inside the "scripts" field under "appium"
+   * inside of the driver/plugins "package.json" file
+   *
+   * @param {string} ext - name of the extension to run a script from
+   * @param {string} scriptName - name of the script to run
+   */
+  async run ({ext, scriptName}) {
+    if (!_.has(this.config.installedExtensions, ext)) {
+      throw new Error(`please install the ${this.type} first`);
+    }
+
+    const extConfig = this.config.installedExtensions[ext];
+
+    if (!_.has(extConfig, 'scripts')) {
+      throw new Error(`The ${this.type} named '${ext}' does not contain the ` +
+                      `"scripts" field underneath the "appium" field in its package.json`);
+    }
+
+    const extScripts = extConfig.scripts;
+
+    if (!_.has(extScripts, scriptName)) {
+      throw new Error(`The ${this.type} named '${ext}' does not support the script: '${scriptName}'`);
+    }
+
+    const runner = new SubProcess(`node`, [extScripts[scriptName]],
```

Comment:
Consider using `process.execPath` instead of `node` as the latter won't work on Windows. See https://stackoverflow.com/questions/49934315/how-to-get-path-of-nodejs-executable-on-runtime